sdk/servicebus/service-bus/test/perf/azure-sb-package/README.md

```diff
@@ -0,0 +1,3 @@
+## Pre-requisites
+
+- Run "npm install" to get "azure-sb" package for the sample.
```

Comment:
This instruction can be included in the comment section on setup instructions in the test sample.
Any reason to have a separate README file?

Reply:
Verify Readmes job failed because of not having a readme file.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=24643
![image](https://user-images.githubusercontent.com/10452642/57052571-05f11000-6c3d-11e9-84c8-6feb597cc017.png)

-----------------------------------

So, the easiest workaround seemed to be adding a dummy readme file and to update the list of files to be ignored in `.docsettings.yml`. 
![image](https://user-images.githubusercontent.com/10452642/57052612-4a7cab80-6c3d-11e9-94df-43758338cfa3.png)

Is there a better way to handle the problem like by not having a readme file?
Maybe @KarishmaGhiya can help?

Reply:
I believe the reason the Verify Readme job would even care about this folder is because of the `package.json` file i.e the rule could be that every `package.json` file needs to be accompanied by a README file

I would say, remove the package.json file (and the README) and just include the instruction to run `npm install azure-sb` in the top comment section of the sample

Reply:
I see some issues here.
- If we just do this - `npm install azure-sb @types/azure-sb`, the package will be installed in the `sdk\servicebus\service-bus\node_modules` and the corresponding package.json will be updated.
Whoever works on this sample should be careful not to commit these files.
- Instead, if we do `npm init -y && npm install azure-sb @types/azure-sb`, a new package.json will be created inside `sdk\servicebus\service-bus\azure-sb-package` along with the `node_modules`. Again, whoever works on this should be careful not to commit these files.

Reply:
Honestly I don't like either options.
What do you think about expecting the person running perf tests to copy the file to a separate location and run it from there?